packages/renderer/MdToHtml/renderMedia.ts

```diff
@@ -31,9 +31,10 @@ export default function(link: Link, options: Options) {
 	}
 
 	if (options.audioPlayerEnabled && resource.mime.indexOf('audio/') === 0) {
+		const escapedAudioMime = escapedMime === 'audio/x-flac' ? 'audio/flac' : escapedMime;
```

Comment:
Please add a comment to explain why it's needed, and link to the GitHub issue

Reply:
I added a comment.